server/athenian/api/__init__.py

```diff
@@ -382,8 +392,23 @@ def main() -> Optional[AthenianApp]:
                              args.precomputed_db,
                              args.persistentdata_db,
                              ),
+        on_startup_callbacks=on_startup_callbacks,
         ui=args.ui, auth0_cls=auth0_cls, kms_cls=kms_cls, cache=cache, slack=slack,
         client_max_size=int(os.getenv("ATHENIAN_MAX_CLIENT_SIZE", 256 * 1024)),
         max_load=float(os.getenv("ATHENIAN_MAX_LOAD", 12)))
     app.run(host=args.host, port=args.port, print=lambda s: log.info("\n" + s))
     return app
+
+
+def _get_preloader(mdb_conn, pdb_conn, log):
+
+    async def _prepare_df_cache(app):
+        log.info("Preparing MemoryCache")
+        mdb, pdb = databases.Database(mdb_conn), databases.Database(pdb_conn)
+        await asyncio.gather(mdb.connect(), pdb.connect())
```

Comment:
Hence we'll not need `asyncio.gather` here. But if we needed it, I'd suggest `athenian.api.utils.gather` instead, exception handling is a bitch.

Reply:
done